Dockerfile.dbg

```diff
@@ -35,13 +34,17 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DEBCONF_NONINTERACTIVE_SEEN true
 
 RUN set -x \
-  && apk update \
-  && apk add ca-certificates \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends ca-certificates wget \
+  && wget https://github.com/nats-io/natscli/releases/download/0.0.26/nats-0.0.26-amd64.deb \
```

Comment:
Download nats-cli in the previous step and copy the binary int this step. Just follow the restic example.